src/interfaces/IScaledBalanceToken.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: agpl-3.0
+pragma solidity >=0.4.22 <0.9.0;
```

Comment:
All tests are the same as ERC20 right?
Maybe we should only keep the ones that are different like the add new token?
To be discussed